src/questions/Questions.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+const Questions = (props) =>{
+    function selectQuestion(e){
```

Comment:
use react state to store opened item, or list of opened items and change the classname then using `classnames` package
https://github.com/meltaway/dsc-studygroup-react/pull/1#discussion_r606844128